README.rst

```diff
@@ -175,6 +175,13 @@ In some cases, for example when using `GitLab's NPM package registry <https://do
             RUN npm config set '//gitlab.example.com/api/v4/projects/<your_project_id>/packages/npm/:_authToken' '<your_token>'
             RUN npm install '@edx/frontend-component-header@npm:@foo/<your_frontend_component_header_name>@latest'
 
+Running in production
+---------------------
+
+When running this plugin in production, in addition to the standard `DNS configuration <https://docs.tutor.overhang.io/install.html#configuring-dns-records>`__ it is necessary for the domain name that corresponds to the ``MFE_HOST`` setting to be properly configured at a registrar.  Once the domain can be resolved at, for example, ``apps.your.domain``, you should run::
```

Comment:
This is not quite right. The DNS config instructions suggest to add a CNAME for all subdomains. This should be enough in the default case but of course it will break in the following cases:

- the users did not read the docs relative to the DNS config (this is very likely as the docs do not make a great job at bubbling up this piece of information). Maybe right below (or in?) the usage instructions?
- the user configures a different `MFE_HOST`.

In any case, the DNS instructions are very important, so they should be higher up in the README. Maybe in the

Reply:
> This is not quite right. The DNS config instructions suggest to add a CNAME for all subdomains.
> the user configures a different MFE_HOST.

Technically, "properly configured at a registrar" is exactly what this means.  After all, as you mention, the user doesn't *have* to use a CNAME if they don't want to, particularly if they have a different `MFE_HOST` set up.  In any case, for consistency, I'll reword it accordingly.

> Maybe right below (or in?) the usage instructions? (...) higher up in the README.  Maybe in the (?)

Your last comment was cut off at the end, but I'll take the previous placement suggestion. 👍🏼